src/User/ProfilePage.test.jsx

```diff
@@ -16,32 +17,135 @@ jest.mock("react-router-dom", () => {
 configure({ adapter: new Adapter() });
 
 describe("ProfilePage", () => {
-    const mockData = {
-        id: '11',
-        username: "RMorey",
-        firstName: "Ross",
-        lastName: "Morey",
-        email: "Ross.Morey@live.co.uk",
-        occupation: "Fun" 
+  const mockDataWithManager = {
+    id: 11,
+    username: "RMorey",
+    firstName: "Ross",
+    lastName: "Morey",
+    email: "Ross.Morey@live.co.uk",
+    occupation: "Fun",
+    managerId: 1
   };
 
-  it("renders profile information", async () => {
-    expect.assertions(5);
+  const mockDataWithoutManager = {
+    id: 11,
+    username: "RMorey",
+    firstName: "Ross",
+    lastName: "Morey",
+    email: "Ross.Morey@live.co.uk",
+    occupation: "Fun",
+    managerId: null
+  };
+
+  const mockManagerName = "Joe Bloggs";
+
+  const mockNoManagerName = "No_Manager";
```

Comment:
Could import this from your constants so if this changes the test won't need an additional change